test/scenarios/attestation/output/src/attestation/azext_attestation/tests/cmdlet/conftest.py

```diff
@@ -0,0 +1,2 @@
+def pytest_runtest_logreport(report):
+    report.nodeid = "..." + report.nodeid[report.nodeid.rfind("::"):]
```

Comment:
wondering who will invoke this file ?

Reply:
This file is used to set pytest output format